packages/pds/src/services/repo/index.ts

```diff
@@ -215,18 +215,20 @@ export class RepoService {
     commitData: CommitData,
     writes: PreparedWrite[],
   ) {
-    // @TODO move to appview
-    writes.map((write) => {
-      if (
-        write.action === WriteOpAction.Create ||
-        write.action === WriteOpAction.Update
-      ) {
-        this.labeler.processRecord(write.uri, write.record)
-      }
-    })
-
     const seqEvt = await sequencer.formatSeqCommit(did, commitData, writes)
     await sequencer.sequenceEvt(this.db, seqEvt)
+
+    // @TODO move to appview
```

Comment:
does this still apply and if so what needs to be done?

Reply:
yeah, but it's part of a larger architectural chnage. tracking it in a few locations with @TODOs